schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/3-0-1

```diff
@@ -0,0 +1,192 @@
+              "additionalProperties": false
+            }
+          ]
+        },
+        "timestamps": {
+          "description": "Set of timestamps associated with the batch",
+          "type": "object",
+          "properties": {
+            "jobStarted": {
+              "description": "Time when the batch started being shredded",
+              "type": "string",
+              "format": "date-time"
+            },
+            "jobCompleted": {
+              "description": "Time when the batch shredding has been finished (and the message being prepared)",
+              "type": "string",
+              "format": "date-time"
+            },
+            "min": {
+              "description": "The earliest collector_stamp available in the batch",
+              "type": ["string", "null"],
+              "format": "date-time"
+            },
+            "max": {
+              "description": "The latest collector_tstamp available in the batch",
+              "type": ["string", "null"],
+              "format": "date-time"
+            }
+          },
+          "required": ["jobStarted", "jobCompleted", "min", "max"]
+        },
+        "processor": {
+          "description": "Identificator of a shredder sent the message",
+          "type": "object",
+          "properties": {
+            "artifact": {
+              "description": "Name of the artifact",
+              "type": "string",
+              "maxLength": 64
+            },
+            "version": {
+              "description": "Semantic Version of the artifact",
+              "type": "string",
+              "maxLength": 16
+            }
+          },
+          "required": ["artifact", "version"]
+        },
+        "count": {
+          "description": "Count of events in the batch, null means the count could not be calculated",
+          "type": ["object", "null"],
+          "properties": {
+            "good": {
+              "description": "Amount of good events in the batch",
+              "type": "integer",
+              "minimum": 0
+            }
+          }
+        }
+      },
+      "required": ["base", "compression", "typesInfo", "timestamps", "processor"],
+      "additionalProperties": false
+    },
+    "application": {
+      "description": "Loader's name and version",
+      "type": "string",
+      "maxLength": 128
+    },
+    "attempt": {
+      "description": "How many attempts were taken to load the batch",
+      "type": "integer",
+      "minimum": 0
+    },
+    "loadingStarted": {
+      "description": "Time when loader started loading (first attempt)",
+      "type": "string",
+      "format": "date-time"
+    },
+    "loadingCompleted": {
+      "description": "Time when loader finished loading (should match the ingestion timestamp in manifest)",
+      "type": "string",
+      "format": "date-time"
+    },
+    "recoveryTableNames": {
+      "description": "List of recovery table names loaded in this batch",
+      "type": "array",
```

Comment:
I think we should put null as extra type (at least decide a policy). In this same schema we have a precedent for count field line 145.

Reply:
@istreeter has stated its not needed so I think we should move to not requiring this and trusting that we do not need to define "null" in optional fields anymore.  This likely also means removing that advice from `iglu` itself given that its not strictly needed and doesn't add anything meaningful?

Reply:
I’m ok for this schema, but as I pointed up we would have the schema adopting two different approaches (line 145). 
> its not needed so I think we should move to not requiring this and trusting that we do not need to define "null" in optional fields anymore.
IMO it's not totally correct because a not required property is a property that doesn't compare in the json payload. If the property compare but it's null it would be invalidated. If we add the null type to the property we are sure that even a null property would be considered accepted.
I don't know if it's ok in this case but I know this was the extra safety adopted for all the schemas developed by Activation to prevent those kind of issues, and probably the reason why it's in `igluctl` as a warning (?!).

Reply:
@istreeter @AlexBenny should I just add the null type for the sake of history and we can have the semantic debate on how this should be ideally going forward at another time?